spec/phase2.html

```diff
@@ -0,0 +1,323 @@
+          // !!!! IMPORTANT !!!! MAKE THE ABOVE BLINK IN YOUR HEAD
+      };
+    </script>
+  </head>
+  <body>
+    <section id="introduction">
+      <h1>Introduction</h1>
+      <p>
+        This document is the spec for Phase 2 of the Accessibility Object
+        Model spec. For background, please refer to the
+        <a href="https://github.com/WICG/aom/blob/master/explainer.md">
+          Accessibility Object Model Explainer</a> for the background and
+        motivation, and then
+        <a href="index.html">Phase 1</a>.
+      </p>
+    </section>
+    <section id="actions">
+      <h2>
+        Phase 2: Accessible Actions
+      </h2>
+        <p>
+          In this phase, we extent AccessibleNode so that it's also an
+          EventTarget.  AccessibleNodes receive
+          special <b>accessibility input events</b> or <b>accessible
+          actions</b> that represent explicit intents to control the
+          user agent via an accessibility API
+        </p>
+        <pre class="idl">
+          interface AccessibleNode : EventTarget {
+            ...
+          };
+        </pre>
+        <p>
+          Accessibility input events are needed for two reasons.
+          First, because it's possible to create AccessibleNodes
+          that don't correspond to a DOM node and authors need to
+          be able to listen to events on these nodes. Second,
+          because there are many types of specific user intents
+          that can be communicated to the user agent via accessibility
+          APIs that don't correspond to existing keyboard and mouse
+          events - for example some assistive technology has
+          keystrokes or gestures to increment a slider, or dismiss
+          a pop-up.
+        </p>
+        <pre class="idl">
+          interface AccessibleSetValueEvent : Event {
+            attribute DOMString value;
+          };
+        </pre>
+        <table>
+          <caption>
+            Accessibility event table
+          </caption>
+          <thead>
+            <tr>
+              <th>event name</th>
+              <th>type</th>
+              <th>valid roles</th>
+              <th>fallback behavior</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><code>"accessibleclick"</code></td>
+              <td><code>AccessibleInputEvent</code></td>
+              <td><em>all roles</em></td>
+              <td>Fire <code>"click"</code> event on DOM node</td>
+            </tr>
+            <tr>
+              <td><code>"accessiblefocus"</code></td>
+              <td><code>AccessibleInputEvent</code></td>
+              <td><em>all roles</em></td>
+              <td>Focus the associated DOM element</td>
+            </tr>
+            <tr>
+              <td><code>"accessiblesetvalue"</code></td>
+              <td><code>AccessibleSetValueEvent</code></td>
+              <td>
+                <code>"scrollbar"</code>,
```

Comment:
Yes, I haven't updated the other phases much yet, I just moved them to
separate files.